pytravharv/common.py

```diff
@@ -0,0 +1,66 @@
+import os
+from logging import getLogger
+from urllib.parse import quote, unquote
+
+import validators
+from pyrdfj2 import J2RDFSyntaxBuilder
+from rdflib import Graph
+
+from pytravharv.WebAccess import fetch
+
+log = getLogger(__name__)
+
+QUERY_BUILDER = J2RDFSyntaxBuilder(
+    templates_folder=os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "templates"
+    )
+)
+
+
+def graph_name_to_uri(graph_name: str) -> str:
+    """
+    Convert a graph name to a URI.
+    """
+    return "uri:PYTRAVHARV:{}".format(quote(graph_name))
+
+
+def uri_to_graph_name(uri: str) -> str:
+    """
+    Convert a URI to a graph name.
+    """
+    length = len("uri:PYTRAVHARV:")
+    return unquote(uri[length:])
+
+
+def insert_resource_into_graph(graph: Graph, resource: str):
```

Comment:
this seems to be part of the concern to initially load some state into the store?

as mentioned before I don't see it useful outside the __main__ context